vaadin-login-flow-parent/vaadin-login-flow/src/main/java/com/vaadin/flow/component/login/AbstractLogin.java

```diff
@@ -267,4 +311,12 @@ public ForgotPasswordEvent(AbstractLogin source, boolean fromClient) {
     public void onEnabledStateChanged(boolean enabled) {
         getElement().setProperty(PROP_DISABLED, !enabled);
     }
+
+    private void checkActionAttributeUsedWithLoginListeners() {
```

Comment:
Nit (optional): This might have a more descriptive name, for example, `warnIfActionAndLoginListenerUsedTogether`

Reply:
Done